build_tools/bazel/lit_test.bzl

```diff
@@ -61,6 +61,7 @@ def lit_test(
         data = None,
         visibility = None,
         env = None,
+        timeout = None,
```

Comment:
Seem like you've got some unrelated diffs here. Could you merge from/rebase on main?